src/azure-cli/azure/cli/command_modules/network/_validators.py

```diff
@@ -263,6 +265,19 @@ def validate_dns_record_type(namespace):
             return
 
 
+def validate_application_gateway_identity(cmd, namespace):
+    from msrestazure.tools import is_valid_resource_id, resource_id
+
+    if namespace.user_assigned_identity and not is_valid_resource_id(namespace.user_assigned_identity):
```

Comment:
If validate failed, what kind of info does service returned?

Reply:
This is a local validation. I don't get your point.